docs/mobile-apps/automated-testing/espresso-xcuitest/xcuitest.md

```diff
@@ -245,6 +245,50 @@ reporters:
     filename: saucectl-report.xml
 ```
 
+---
+### `json`
+<p><small>| OPTIONAL | OBJECT |</small></p>
+
+The JSON reporter gathers test results from all jobs and combines them into a single report.
+
+```yaml
+reporters:
+  json:
+    enabled: true
+    filename: saucectl-report.json
+    webhookURL: https://my-webhook-url
+```
+
+---
+#### `enabled`
+<p><small>| OPTIONAL | BOOLEAN |</small></p>
+
+Toggles the reporter on/off.
+
+```yaml
+    enabled: true
+```
+
+---
+#### `webhookURL`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+Specifies the webhook URL. When saucectl test is finished, it'll send an HTTP POST with a JSON payload to the configured webhook URL.
+
+```yaml
+    webhookURL: https://my-webhook-url
+```
+
+---
+#### `filename`
```

Comment:
Per our style, these all need to be H3 so that they show up in the right-hand nav.

Reply:
Hey @kimsauce , I'm just wondering how to add these fields with the inheriting orders. enabled, webhookURL and filename are all under json fields. If I set them all as H3, will it break the inheriting orders in the nav?